### ISSUE TITLE
since 4.0.4:  no more thumbnail_src for hastags??

### ISSUE THREAD

**Issue by dcreekp** [instaloader/instaloader#137]
Thank you for the work put into this library:)

In 4.0.3 we used to be able to access `['node']['thumbnail_src']` from the json files returned by a hashtag query.
In version 4.0.4,  json files returned by a profile query have `['node']['thumbnail_src']` but json files from a hashtag query no longer provide a key for thumbnail_src. 

Is this deliberate? Am I missing something?

I'm raising this issue as the release note for 4.0.4 says:
"Fixes detection of sidecars and videos when downloading hashtags (#133)" but doesn't mention anything about thumbnails.

**Comment by aandergr** [comment]
Thanks for reporting this! I fixed it in commit 969d4e3. To update:
```
$ pip3 install --upgrade git+https://github.com/instaloader/instaloader
```

**Comment by dcreekp** [comment]
Awesome, thanks very much!
